bin/compile

```diff
@@ -137,6 +135,10 @@ for PACKAGE in $PACKAGES; do
   fi
 done
 
+aws s3api get-object --bucket $ENV_DIR/S3_BUCKET_URL --key chrome/google-chrome-stable_${chrome_version}.deb $CACHE_DIR/google-chrome.deb --region $ENV_DIR/S3_REGION
+
+dpkg -i $CACHE_DIR/google_chrome.deb
```

Comment:
Can you remove this `dpkg -i` sentence and modify the `aws s3api` call to download the file to `$APT_CACHE_DIR/archives/` instead of `$CACHE_DIR/google-chrome.deb` please?

That way the script will extract the chrome package in the same way it used to do before, inside the for loop that's on line 144:

![image](https://github.com/user-attachments/assets/dfec7805-6cfb-4386-a0fe-82a1d904a74e)

Just to follow what we know works and reuse existing code. Plus the existing logline is really useful too.